pkg/collector/energy/node_energy_collector.go

```diff
@@ -66,10 +66,14 @@ func UpdateNodeComponentsEnergy(nodeStats *stats.NodeStats, wg *sync.WaitGroup)
 // UpdateNodeGPUEnergy updates each GPU power consumption. Right now we don't support other types of accelerators
 func UpdateNodeGPUEnergy(nodeStats *stats.NodeStats, wg *sync.WaitGroup) {
 	defer wg.Done()
-	if config.EnabledGPU && gpu.IsGPUCollectionSupported() {
-		gpuEnergy := gpu.GetAbsEnergyFromGPU()
-		for gpu, energy := range gpuEnergy {
-			nodeStats.EnergyUsage[config.AbsEnergyInGPU].SetDeltaStat(fmt.Sprintf("%d", gpu), uint64(energy))
+	if config.EnabledGPU {
+		if gpus, err := acc.Registry().ActiveAcceleratorsByType(acc.GPU); err == nil {
+			for _, g := range gpus {
```

Comment:
Nearly all calls to ActiveAccelerators use `acc.GPU`. this makes me feel that perhaps dummy shouldn't an accelerator type. Is a Dummy type different to `GPU` in any the registry treats it?

Reply:
There was also QAT, which I plan on readding after I reimplement it as The current implementation will not work on RHEL. 

There will be other devices I'm sure in the future also DPUs and IPUs... we are just starting with fixing up what's currently supported in kepler